content/blog/2020-10-26-october-20-community-gems.md

```diff
@@ -0,0 +1,163 @@
+---
+title: October '20 Community Gems
+date: 2020-10-26
+description: |
+  A roundup of technical Q&A's from the DVC community. This month, learn how
+  DVC files work, how to use DVC plots for multi-class classification problems,
+  and how to deal with some spooky error messages 👻.
+descriptionLong: |
+  A roundup of technical Q&A's from the DVC community. This month, learn how
+  DVC files work, how to use DVC plots for multi-class classification problems,
+  and how to deal with some spooky error messages 👻.
+picture: 2020-10-26/Gems_Oct_20.png
+author: elle_obrien
+commentsUrl: https://discuss.dvc.org/t/october-20-community-gems/535
+tags:
+  - Discord
+  - Gems
+  - CML
+  - Vega
+  - Metrics
+---
+
+## DVC questions
+
+### [Q: What's in a `.dvc` file, and what would happen if decided not push my `.dvc` files to my Git repo?](https://discordapp.com/channels/485586884165107732/485596304961962003/760920403064520755)
+
+DVC creates lightweight metafiles (`.dvc` files) that correspond to large
+artifacts in your project. These `.dvc` files contain pointers to your artifacts
+in remote storage (we use a simple content-based storage scheme). Because we use
+content-based storage, the remote storage itself isn't easy to browse (although
+[there are some discussions](https://github.com/iterative/dvc/issues/3621) about
+how to make the content more "discoverable").
+
+Your `.dvc` files are needed to establish meaningful links between
+human-readable filenames and file contents in remote storage, as well as to use
+Git versioning on your stored datasets and models. You can think of your DVC
+remote storage as a _compliment_ to your Git repository, not a replacement.
+
+In other words... if you're not Git versioning your `.dvc` files, you're not
```

Comment:
this can a little bit misleading (and can sound too scary)... if we still save artifacts there are other indirect (manual and less reliable) ways to get them back (analyze content, for example, or timestamps). It's like losing your keys - there are other way to get in, but way more complicated.

Reply:
i brought these ways up and toned down the langauge so it's more clear losing your `.dvc` files doesn't means the end of the world